### PR TITLE
[DOTSLASH] Prioritize executable files

### PR DESCRIPTION
**Pull request by Monkeyank**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Prioritizes executable files
**What is the current behavior? (You can also link to an open issue here)**
#220 No priority given to executable files.
**What is the new behavior (if this is a feature change)?**
Prioritizes executable files
**Additional info:**

### CODE REVIEW

**Comment by QuiiBz** [comment]
Looks like a `@` should be removed at the end of some files, as you can see:
Screenshot 2021-06-12 at 13 24 20

**Comment by Monkeyank** [comment]
> Looks like a `@` should be removed at the end of some files, as you can see:
> Screenshot 2021-06-12 at 13 24 20

I removed the other characters that are present from `ls -F`, like `"@", "%", "|"`

**Comment by QuiiBz** [comment]
I'm not sure if that would not be better to directly make this feature in the Fig app, because it can be used by other specs. @cstrnt @falky97 any thought on this?

File: dev/dotslash.ts [hunk 1]
```diff
@@ -1,10 +1,55 @@
...
+      const isFolder = path.endsWith("/");
+
+      if (isFolder) {
+        filename = components[components.length - 2] + "/";
+      }
+
+      // Removes character at the end of the filepath
+      if (path.endsWith("*" || "@" || "%" || "|")) {
+        filename = filename.slice(0, -1);
+      }
```
Comment (QuiiBz) [inline dev/dotslash.ts:34]: Still having the `@` at the end - the others `OR` will never be called in this code. You can either duplicate `path.endsWith("<symbol>")` for each symbol, or grab the last letter and check if an array of all the symbols contains it. That would be something like:
```js
if (["*", "*", "%", "|"].includes(lastLetter)) {
...
```

Screenshot of the `@`:
Screenshot 2021-06-24 at 09 05 57

**Comment by mattschrage** [comment]
> I'm not sure if that would not be better to directly make this feature in the Fig app, because it can be used by other specs. @cstrnt @falky97 any thought on this?

We could potentially include this as a new template called `executables` but I think for the time being, it is fine for this to be defined here.

**Review by mattschrage — APPROVED** [review]
I'm excited for this one to land in production! Great work @Monkeyank 🙌

**Review by QuiiBz — APPROVED** [review]
This is going to be really helpful, thanks a lot @Monkeyank!

**Pull request merged**